src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -12,30 +12,23 @@ export interface VenuePartygoersProps {
 export const VenuePartygoers: React.FC<VenuePartygoersProps> = ({
   venueId,
 }) => {
-  const { isLoading, parentVenue, currentVenue } = useRelatedVenues({
+  const { isLoading } = useRelatedVenues({
     currentVenueId: venueId,
   });
 
   const { recentWorldUsers, isRecentWorldUsersLoaded } = useRecentWorldUsers();
 
   const numberOfRecentWorldUsers = recentWorldUsers.length;
 
-  const title = useMemo<string>(() => {
-    if (isLoading || !isRecentWorldUsersLoaded) return "";
+  const userCount = useMemo<number>(() => {
+    if (isLoading || !isRecentWorldUsersLoaded) return 0;
 
-    const attendeesTitle =
-      parentVenue?.attendeesTitle ??
-      currentVenue?.attendeesTitle ??
-      "attendees";
+    return numberOfRecentWorldUsers;
+  }, [isLoading, isRecentWorldUsersLoaded, numberOfRecentWorldUsers]);
 
-    return `${numberOfRecentWorldUsers} ${attendeesTitle} online`;
-  }, [
-    isLoading,
-    isRecentWorldUsersLoaded,
-    parentVenue?.attendeesTitle,
-    currentVenue?.attendeesTitle,
-    numberOfRecentWorldUsers,
-  ]);
-
-  return <div className="venue-partygoers-container">{title}</div>;
+  return (
+    <div className="VenuePartygoers__container">
+      <span className="VenuePartygoers__number">{userCount}</span> burners here
```

Comment:
`VenuePartygoers__number` -> `VenuePartygoers__users-count` to keep in line with the variable?

Reply:
fixed 433dfe8